styles/_my-activities.scss

```diff
@@ -109,21 +109,21 @@
       width: 100%;
     }
 
-    .organization-subscribe,
-    .organization-unsubscribe {
+    .organization-subscribe {
       @include tablet {
         margin-left: 10px;
         [dir="rtl"] & { margin: 0 10px 0 0; }
       }
 
-      line-height: 40px;
+      button {
+        @extend .button;
+        line-height: 40px;
       max-height: 40px;
       padding: 0 20px;
-    }
-
-    .organization-unsubscribe {
-      background-color: $button-color;
-      color: $brand_text_color;
+        &[data-selected="true"] {
```

Comment:
Are we changing these to `aria-pressed` later?

Reply:
I thought we were discussing `aria-pressed` for votes only.
These changes are related to the follow button/menu-button. When the state changes the label changes as well so not sure `aria-pressed` is appropriate (?) also due to the menu behaviour.

Reply:
Ah yes, sorry, I'm mixing things up here. Excuse my fried egg brain 🍳

Reply:
No worries! Same here 🍳